### PR TITLE
add tablestructure

### PR DESCRIPTION
**Pull request by MarcRaeder**
add the BASIC structure of the table

### CODE REVIEW

**Comment by stefanluth** [comment]
`table.js` is empty, and no PR description :)

File: table.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.table {
+  border: 1px solid black;
+  width: 100%;
+  height: 100%;
+}
+
+.table__headerRow {
```
Comment (DevBastiS) [inline table.css]:
```suggestion
.table__header-row {
```

Apply this to all css classes please.

File: table.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <tr class="table__contentRow">
+        <td class="table-contentRow__body">Peter</td>
+        <td class="table-contentRow__body">50</td>
+        <td class="table-contentRow__body">USA</td>
+        <td class="table-contentRow__body">Green</td>
+      </tr>
+    </table>
+  </body>
+  <script> scr = "table.js"</script>
+
```
Comment (DevBastiS) [inline table.html]:
```suggestion
```

File: table.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (DevBastiS) [inline table.html]:
```suggestion
```

**Review by DevBastiS — CHANGES_REQUESTED** [review]
Also what @stefanluth said.

File: table.css [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  text-align: center;
+}
+
+.table__headerRow-body {
+  border: 1px solid black;
+}
+
+.table__contentRow-body {
+  border: 1px solid black;
+}
```
Comment (DevBastiS) [inline table.css]:
```suggestion
.table__header-row {
  background-color: goldenrod;
  height: 50px;
}

.table__content-row {
  border: 1px solid black;
  text-align: center;
}

.table__header-row-body {
  border: 1px solid black;
}

.table__content-row-body {
  border: 1px solid black;
}
```

**Review by DevBastiS — CHANGES_REQUESTED** [review]

**Comment by MarcRaeder** [comment]
> `table.js` is empty, and no PR description :)

yes but is only the BASIC Structure. And this includes the .js file

**Review by stefanluth — APPROVED** [review]

File: table.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      </tr>
+      <tr class="table__content-row">
+        <td class="table__content-row-body">Peter</td>
+        <td class="table__content-row-body">50</td>
+        <td class="table__content-row-body">USA</td>
+        <td class="table__content-row-body">Green</td>
+      </tr>
+    </table>
+  </body>
+  <script> scr = "table.js"</script>
```
Comment (DevBastiS) [inline table.html]:
```suggestion
  <script> scr="table.js"</script>
```

File: table.html [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        <th class="table__header-row-body">FavouriteColor</th>
+      </tr>
+      <tr class="table__content-row">
+        <td class="table__content-row-body">Marc</td>
+        <td class="table__content-row-body">21</td>
+        <td class="table__content-row-body">Germany</td>
+        <td class="table__content-row-body">Blue</td>
+      </tr>
+      <tr class="table__content-row">
+        <td class="table__content-row-body">Bob</td>
```
Comment (DevBastiS) [inline table.html]: Sorry I should have seen that earlier:
```suggestion
        <td class="table__content-row-cell">Bob</td>
```

Comment (DevBastiS) [reply table.html]: This is a cell of the table not a body. You could argue that "table__cell" would suffice too.

**Review by DevBastiS — APPROVED** [review]
Please fix these changes then merge.

**Pull request merged**